concepts/array-analysis/introduction.md

```diff
@@ -1,6 +1,7 @@
 # Introduction
 
```

Comment:
General comment: It is important that the about.md includes all the content from the introduction because the student can only see the about content after completing the exercise. So it would be great if you could have a look how to ensure that while keeping any advanced content we currently have in about.md.